src/Sarif/Readers/DateTimeConverter.cs

```diff
@@ -27,13 +27,15 @@ public override bool CanConvert(Type objectType)
 
         public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
         {
+            if (reader.Value is DateTime) { return reader.Value; }
```

Comment:
JSON.NET appears to have some built-in DateTime detection so, at least in the current test cases, the object is already converted. We still need the converter, to output the correct format. But I haven't yet confirmed whether this conditional is always satisfied (in which case we can simplify the implementation to simply return reader.Value. Will follow-up